src/main/java/org/broadinstitute/hellbender/tools/pon/allelic/AllelicPanelOfNormals.java

```diff
@@ -0,0 +1,331 @@
+                throw new UserException.BadInput("MLE global hyperparameter values for allelic panel of normals were not specified correctly in comment lines of file " + inputFile.getAbsolutePath() + ".");
+            }
+            final double globalAlpha = Double.valueOf(globalAlphaCommentLines.get(0).replace(TableUtils.COMMENT_PREFIX + GLOBAL_ALPHA_COMMENT_STRING, ""));
+            final double globalBeta = Double.valueOf(globalBetaCommentLines.get(0).replace(TableUtils.COMMENT_PREFIX + GLOBAL_BETA_COMMENT_STRING, ""));
+            ParamUtils.isPositive(globalAlpha, "MLE global hyperparameter alpha must be positive.");
+            ParamUtils.isPositive(globalBeta, "MLE global hyperparameter alpha must be positive.");
+            return new HyperparameterValues(globalAlpha, globalBeta);
+        } catch (final IOException | UncheckedIOException e) {
+            throw new UserException.CouldNotReadInputFile(inputFile, e);
+        } catch (final NumberFormatException e) {
+            throw new UserException.BadInput("MLE global hyperparameter values for allelic panel of normals were not specified correctly in comment lines of file " + inputFile.getAbsolutePath() + ".");
+        }
+    }
+
+    private void initializeSiteToHyperparameterPairMap(final AllelicCountCollection counts) {
+        logger.info("Initializing allelic panel of normals...");
+        for (final AllelicCount count : counts.getCounts()) {
+            final SimpleInterval site = count.getInterval();
+            final HyperparameterValues hyperparameterValues = new HyperparameterValues(
+                    globalHyperparameterValues.alpha, globalHyperparameterValues.beta,
+                    count.getAltReadCount(), count.getRefReadCount());
+            if (siteToHyperparameterValuesMap.containsKey(site)) {
+                throw new UserException.BadInput("Input AllelicCountCollection for allelic panel of normals contains duplicate sites.");
+            } else {
+                siteToHyperparameterValuesMap.put(site, hyperparameterValues);
+            }
+        }
+        logger.info("Allelic panel of normals initialized.");
+    }
+
+    private void throwExceptionIfPoNIsEmpty() {
+        if (this.equals(EMPTY_PON)) {
+            throw new UnsupportedOperationException("Cannot get MLE hyperparameters for empty panel of normals.");
+        }
+    }
+
+    //TODO SL use methods in David's PR
+    private static double meanBias(final double alpha, final double beta) {
+        return alpha / beta;
+    }
+
+    private static double biasVariance(final double alpha, final double beta) {
+        return alpha / (beta * beta);
+    }
+
+    //returns a list of the sites in siteToHyperarameterPairMap that are sorted by SimpleInterval (contigs are sorted by lexicographical order)
+    private static List<Map.Entry<SimpleInterval, HyperparameterValues>> collectSortedMapEntries(final Map<SimpleInterval, HyperparameterValues> siteToHyperparameterPairMap) {
```

Comment:
Should this be line-wrapped? What is the policy we use for line-wrapping in general?